tensorflow/python/ops/image_ops_impl.py

```diff
@@ -308,7 +308,7 @@ def fix_image_flip_shape(image, result):
     result: flipped or transformed image
 
   Returns:
-    An image whose shape is at least None,None,None.
+    An image whose shape is at least None, None, None.
```

Comment:
Should be as a tuple, `(None, None, None)`.

Reply:
made required changes